opus/application/static_media/js/widgets.js

```diff
@@ -85,17 +104,300 @@ var o_widgets = {
             let minVal = $(e.currentTarget).data("min");
             let maxVal = $(e.currentTarget).data("max");
             let widgetId = $(e.currentTarget).data("widget");
+            let minInputSlug = $(e.currentTarget).parent(".container").attr("data-mininput");
 
             // NOTE: We need support both RANGE & STRING inputs, for now we implement RANGE first.
             if ($(`#${widgetId} input.RANGE`).length !== 0) {
-                o_widgets.fillRangesInputs(widgetId, maxVal, minVal);
+                o_widgets.fillRangesInputs(widgetId, minInputSlug, maxVal, minVal);
                 // close dropdown and trigger the search
-                $(`#${widgetId} input.op-range-input-min`).dropdown("toggle");
-                $(`#${widgetId} input.RANGE`).trigger("change");
+                $(`#${widgetId} input.op-range-input-min[name="${minInputSlug}"]`).dropdown("toggle");
+                $(`#${widgetId} input.RANGE[name="${minInputSlug}"]`).trigger("change");
+
+                let minInput = $(`#${widgetId} input.RANGE[name="${minInputSlug}"]`);
+                let slug = o_utils.getSlugOrDataWithoutCounter(minInputSlug);
+                let slugName = minInput.attr("data-slugname");
+                let uniqueid = minInput.attr("data-uniqueid");
+                let oppositeSuffixSlug = (slug.match(/(.*)1$/) ? `${slugName}2` : `${slugName}1`);
+                $(`#${widgetId} input.RANGE[name*="${oppositeSuffixSlug}"][data-uniqueid="${uniqueid}"]`)
+                .trigger("change");
```

Comment:
I think doing multiple line expressions like this in JS is dangerous because JS will randomly decide the first line should be ended with a virtual ";". Better to just leave on one line.